android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/viewshot/RNViewShotModule.java

```diff
@@ -84,16 +81,17 @@ public void captureRef(int tag, ReadableMap options, Promise promise) {
                 : Formats.PNG;
 
         final double quality = options.getDouble("quality");
-        final Integer scaleWidth = options.hasKey("width") ? (int) (dm.density * options.getDouble("width")) : null;
-        final Integer scaleHeight = options.hasKey("height") ? (int) (dm.density * options.getDouble("height")) : null;
+        final Integer scaleWidth = options.hasKey("width") ? options.getInt("width") : null;
+        final Integer scaleHeight = options.hasKey("height") ? options.getInt("height") : null;
         final String resultStreamFormat = options.getString("result");
         final String fileName = options.hasKey("fileName") ? options.getString("fileName") : null;
         final Boolean snapshotContentContainer = options.getBoolean("snapshotContentContainer");
+        final boolean handleGLSurfaceView = options.hasKey("handleGLSurfaceViewOnAndroid") && options.getBoolean("handleGLSurfaceViewOnAndroid");
 
         try {
             File outputFile = null;
             if (Results.TEMP_FILE.equals(resultStreamFormat)) {
-                outputFile = createTempFile(mScopedContext, extension, fileName);
+                outputFile = createTempFile(getReactApplicationContext(), extension, fileName);
```

Comment:
I'm afraid that we still need to use the scoped context, @lukmccall may have more knowledge why

Reply:
Yes, we have to use the scoped context here.  Otherwise, users in expo go won't have access to the outputted file - it won't be saved to the experience temp directory.

Reply:
Fixed